src/main/resources/io/jenkins/plugins/prism/SourceCodeViewModel/index.jelly

```diff
@@ -1,13 +1,21 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:bs="/bootstrap5">
+<j:jelly xmlns:p="/lib/prism" xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:bs="/bootstrap5">
 
   <st:header name="Content-Type" value="text/html;charset=UTF-8"/>
 
   <bs:page it="${it}" notitle="true">
 
-    <st:adjunct includes="io.jenkins.plugins.prism"/>
+    <p:prism prismConfiguration="${it.prismConfiguration}" />
```

Comment:
need to file a PR to warnings NG for this

Reply:
actually it's in this plugin, resolved in https://github.com/jenkinsci/prism-api-plugin/pull/41/commits/66dcb92a0ef09d7bb84de42c7f7c62cd965b0b2b